webview-ui/src/Kaito/Kaito.tsx

```diff
@@ -27,7 +27,7 @@ export function Kaito(initialState: InitialState) {
         <>
             <div className={styles.container}>
                 <h2>Kubernetes AI Toolchain Operator (KAITO) - {state.clusterName}</h2>
-                <VSCodeDivider />
+                <hr />
```

Comment:
I think that there is something to learn here with regards to (not) using react components to replace a built-in html "hr" element... :)